addons/pkg/constants/constants.go

```diff
@@ -169,6 +169,18 @@ const (
 
 	// PackageInstallSyncPeriod is the sync period for kapp-controller to periodically reconcile a PackageInstall
 	PackageInstallSyncPeriod = time.Minute * 10
+
+	// WebhookCertDir is the directory where the certificate and key are stored for webhook server TLS handshake
+	WebhookCertDir = "/tmp/k8s-webhook-server/serving-certs"
```

Comment:
So your original directory matches the default https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/webhook/server.go#L113, if we change this we need to set the config. 

Best practice is to set the configs instead of relying on controller-runtime defaults

Reply:
Fixed